ios/ble/peripheral/Characteristics.swift

```diff
@@ -80,8 +66,8 @@ enum NotificationEvent: String {
     case HANDLE_TRANSMISSION_REPORT = "HANDLE_TRANSMISSION_REPORT"
     case INIT_RESPONSE_CHUNK_TRANSFER = "INIT_RESPONSE_CHUNK_TRANSFER"
     case VERIFICATION_STATUS_RESPONSE = "VERIFICATION_STATUS_RESPONSE"
-    case CONNECTION_STATUS_CHANGE = "CONNECTION_STATUS_CHANGE"
-    
+    case DISCONNECT_STATUS_CHANGE = "DISCONNECT_STATUS_CHANGE"
```

Comment:
I think, It makes sense to keep the name as CONNECTION_STATUS_CHANGE for the event rather than DISCONNECT_STATUS_CHANGE.

Reply:
Discussed with krishna and decided to keep it as DISCONNECT_STATUS_CHANGE